app/templates/staff/user.hbs

```diff
@@ -26,7 +26,7 @@
                             <span class="hidden">User Settings</span>
                         </span>
                     </GhDropdownButton>
-                    <GhDropdown @name="user-actions-menu" @tagName="ul" @classNames="user-actions-menu dropdown-menu dropdown-triangle-top-right">
+                    <GhDropdown @name="user-actions-menu" @tagName="ul" @classNames="user-actions-menu dropdown-menu dropdown-align-right">
```

Comment:
@peterzimon just seen this file changed in the PR. Think it should go in as separate commit straight to master if we do need to change it.